src/lib/components/gadgets/timer/main.svelte

```diff
@@ -15,8 +15,6 @@
 
 	let beeping = $state(false);
 	$effect(() => {
-		if (query) {
```

Comment:
This needs to exist, but I see it's creating a linting error, to fix that set it to this:
```js
$effect(() => {
    query; // Re-run effect when query changes.
    beeping = false;
});
```

Reply:
i forgot svelte reactivity is static-analysis-based...